### PR TITLE
feat: add client-side death sync

### LINKED ISSUE #385: Add client-side death visual for dying player
Depends on #384 

The `isDead` parameter, when changed, is sent to the client. It is necessary to push the player character using `pushActorAway` when the parameter becomes `true`. Player character should start to rise from the floor when the parameter becomes `false`

### PR DESCRIPTION
**Pull request by kkEngine**
closes #385, #386

You can kill if you want
If you want it you will kill
Oh, come on, kill with push
For a property death state

### CODE REVIEW

File: skymp5-client/src/remoteServer.ts [hunk 1]
```diff
@@ -581,13 +597,16 @@ export class RemoteServer implements MsgHandler, ModelSource, SendTarget {
   }
 
   send(msg: Record<string, unknown>, reliable: boolean): void {
+    if (!msg) return;
```
Comment (Pospelove) [inline skymp5-client/src/remoteServer.ts:600]: Excess null check

File: skymp5-client/src/skympClient.ts [hunk 2]
```diff
@@ -57,7 +59,7 @@ const handleMessage = (msgAny: AnyMessage, handler_: MsgHandler) => {
   if (msgType === "hostStart") {
     const msg = msgAny as HostStartMessage;
     const target = msg.target;
-    printConsole("hostStart", target.toString(16));
+    //printConsole("hostStart", target.toString(16));
```
Comment (Pospelove) [inline skymp5-client/src/skympClient.ts:62]: `hostStart` is a really bad log message. I'm glad you removed it but please also remove the commented version of this log message.

File: skymp5-client/src/view.ts [hunk 3]
```diff
@@ -320,7 +321,7 @@ export class FormView implements View<FormModel> {
         );
         if (model.appearance && model.appearance.name)
           refr.setDisplayName("" + model.appearance.name, true);
-        Actor.from(refr)?.setActorValue("attackDamageMult", 0);
+        //Actor.from(refr)?.setActorValue("attackDamageMult", 0);
```
Comment (Pospelove) [inline skymp5-client/src/view.ts:324]: you know what to do

File: skymp5-client/src/view.ts [hunk 4]
```diff
@@ -494,7 +495,7 @@ export class FormView implements View<FormModel> {
           const remoteId = this.remoteRefrId;
           if (ac && ac.is3DLoaded()) {
             tryHostIfNeed(ac, remoteId as number);
-            printConsole("try to rehost");
+            //printConsole("try to rehost");
```
Comment (Pospelove) [inline skymp5-client/src/view.ts:498]: please remove this message at all

File: skymp5-client/webpack.config.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
 const outDirPath = path.resolve(__dirname, "../build/dist/client/Data/Platform/Plugins");
+//const outDirPath = path.resolve("F:/Steam/steamapps/common/Skyrim Special Edition/Data/Platform/Plugins");
```
Comment (Pospelove) [inline skymp5-client/webpack.config.js:4]: Please fix path

**Review by Pospelove — CHANGES_REQUESTED** [review]

File: skymp5-client/src/animation.ts [hunk 6]
```diff
@@ -11,6 +11,25 @@ import {
 import { Movement } from "./movement";
 import { applyWeapDrawn } from "./movementApply";
 
+export enum AnimationEventName {
+  ragdoll = "Ragdoll",
```
[the remark this answers is not in the document]
Comment (kkEngine) [reply skymp5-client/src/animation.ts:15]: > And also remove unused enum constants.
I added this for possible future use

File: skymp5-client/src/animation.ts [hunk 7]
```diff
@@ -11,6 +11,25 @@ import {
 import { Movement } from "./movement";
 import { applyWeapDrawn } from "./movementApply";
 
+export enum AnimationEventName {
```
Comment (Pospelove) [inline skymp5-client/src/animation.ts:14]: plz remove unused constants

**Review by Pospelove — CHANGES_REQUESTED** [review]

File: skymp5-client/src/view.ts [hunk 8]
```diff
@@ -510,7 +510,20 @@ export class FormView implements View<FormModel> {
           if (forcedWeapDrawn === true || forcedWeapDrawn === false) {
             model.movement.isWeapDrawn = forcedWeapDrawn;
           }
-          applyMovement(refr, model.movement);
+          try {
+            applyMovement(refr, model.movement);
+          }
+          catch (e) {
```
Comment (nic11) [inline skymp5-client/src/view.ts:516]: Does this follow styleguide?

**Review by Pospelove — APPROVED** [review]

**Pull request merged**